docs/overview.md

```diff
@@ -1,6 +1,35 @@
-# Overview
+page_title: Registry Service 2.0
+page_description: Explains how to deploy a registry service
+page_keywords: registry, service, images, repository
 
-**TODO(stevvooe):** Table of contents.
+# Registry Service 2.0
 
-**TODO(stevvooe):** Include a full overview of each component and dispatch the
-user to the correct documentation.
+**TODO(stevvooe):** Include a full overview of each component and dispatch the user to the correct documentation.
+
+
+
+### Upgrading
+
+The Docker Registry Service 2.0 is backward compatible with images created by the 1.0 specification. If you are migrating a private registry to version 2.0, you should use the following process:
```

Comment:
I'm not sure this statement is accurate in this context. This implies that there will be effortless compatibility. However, they don't share a backend layout and images will have to be migrated.